src/proto/Stellar.proto

```diff
@@ -3,6 +3,38 @@ syntax = "proto3";
 package TW.Stellar.Proto;
 option java_package = "wallet.core.jni.proto";
 
+message Asset {
+    // Optional in case of non-native asset; the asset issuer address
+    string issuer = 1;
+
+    // Optional in case of non-native asset; the asset alphanum4 code.  Alphanum4 or alphanum12 has to be set.
+    string alphanum4 = 2;
```

Comment:
let's just use 4 symbols for now

Reply:
simplified

Reply:
I suggest to use generic `id` instead of `alphanum4`, if we add alphanum12 support, we don't need to change proto (just check length inside cpp code)